### PR TITLE
add adopter

### PR DESCRIPTION
**Pull request by Planck0591**
add adopter

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

**Review by andreyvelich — COMMENTED** [review]
Thank you for using Katib @Planck0591!

I left one comment.

File: ADOPTERS.md [hunk 1]
```diff
@@ -13,3 +13,4 @@ Please keep the list in alphabetical order.
 | [cisco](https://cisco.com/) |[@ramdootp](https://github.com/ramdootp) | Hyperparameter tuning for conversational AI interface using Rasa |
 | [cubonacci](https://www.cubonacci.com) |[@janvdvegt](https://github.com/janvdvegt) | Hyperparameter tuning within the Cubonacci machine learning platform |
 | [karrot](https://uk.karrotmarket.com/) |[@muik](https://github.com/muik) | Hyperparameter tuning in Karrot ML Platform |
+| [fuzhi](http://www.fuzhi.ai/) | [@planck0591](https://github.com/planck0591) | Experiment and Trial in autoML Platform |
```
Comment (andreyvelich) [inline ADOPTERS.md:16]: Please keep the list in alphabetical order.

```suggestion
| [fuzhi](http://www.fuzhi.ai/) | [@planck0591](https://github.com/planck0591) | Experiment and Trial in autoML Platform |
| [karrot](https://uk.karrotmarket.com/) |[@muik](https://github.com/muik) | Hyperparameter tuning in Karrot ML Platform |
```

**Comment by gaocegege** [comment]
/lgtm

Thanks for your contribution! :tada: :+1:

**Pull request merged**